### PR TITLE
8307193: Several Swing jtreg tests use class.forName on L&F classes

### PR DESCRIPTION
**Pull request by prsadhuk**
Opensource couple RTL tests

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307193](https://bugs.openjdk.org/browse/JDK-8307193): Several Swing jtreg tests use class.forName on L&amp;F classes (**Bug** - P4)

### Reviewers
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - Committer) ⚠️ Review applies to [e9c1da0e](https://git.openjdk.org/jdk/pull/19341/files/e9c1da0e496281832b5c81aa1bce1ed13d9a8eea)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19341/head:pull/19341` \
`$ git checkout pull/19341`

Update a local copy of the PR: \
`$ git checkout pull/19341` \
`$ git pull https://git.openjdk.org/jdk.git pull/19341/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19341`

View PR using the GUI difftool: \
`$ git pr show -t 19341`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19341.diff">https://git.openjdk.org/jdk/pull/19341.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19341#issuecomment-2123910206)

### CODE REVIEW

File: test/jdk/javax/swing/JToolBar/RightLeftOrientation.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            rb.addActionListener(plafChanger);
+            group.add(rb);
+            panel.add(rb);
+            if (i == 0) {
+                rb.setSelected(true);
+            }
+        }
+
+        frame.add(panel);
+
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JToolBar/RightLeftOrientation.java:107]: Blank lines can be removed.

File: test/jdk/javax/swing/JMenuBar/RightLeftOrientation.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            rb.addActionListener(plafChanger);
+            group.add(rb);
+            panel.add(rb);
+            if (i == 0) {
+                rb.setSelected(true);
+            }
+        }
+
+        frame.add(panel);
+
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JMenuBar/RightLeftOrientation.java:102]: Blank lines can be removed.

File: test/jdk/javax/swing/JToolBar/RightLeftOrientation.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/*
+ * @test
+ * @bug 4214514
+ * @summary
+ *     This test checks if tool bars lay out correctly when their
+ *     ComponentOrientation property is set to RIGHT_TO_LEFT.  This test is
+ *     manual.  The tester is asked to compare left-to-right and
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JToolBar/RightLeftOrientation.java:29]:
```suggestion
 *     ComponentOrientation property is set to RIGHT_TO_LEFT. This test is
 *     manual. The tester is asked to compare left-to-right and
```

File: test/jdk/javax/swing/JMenuBar/RightLeftOrientation.java [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+ * questions.
+ */
+/*
+ * @test
+ * @bug 4211731 4214512
+ * @summary
+ *     This test checks if menu bars lay out correctly when their
+ *     ComponentOrientation property is set to RIGHT_TO_LEFT.  This test is
+ *     manual.  The tester is asked to compare left-to-right and
+ *     right-to-left menu bars and judge whether they are mirror images of each
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JMenuBar/RightLeftOrientation.java:30]:
```suggestion
 *     ComponentOrientation property is set to RIGHT_TO_LEFT. This test is
 *     manual. The tester is asked to compare left-to-right and
 *     right-to-left menu bars and judge whether they are mirror images of each
```

File: test/jdk/javax/swing/JToolBar/RightLeftOrientation.java [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+        }
+
+        frame.add(panel);
+
+
+        ltrFrame = new JFrame("Left To Right");
+        Container contentPane = ltrFrame.getContentPane();
+        contentPane.setLayout(new BorderLayout());
+        panel = new JPanel();
+        panel.setBackground( Color.white );
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JToolBar/RightLeftOrientation.java:113]:
```suggestion
        panel.setBackground(Color.white);
```

File: test/jdk/javax/swing/JToolBar/RightLeftOrientation.java [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+                        createToolBar(ComponentOrientation.LEFT_TO_RIGHT));
+        ltrFrame.setSize(400, 140);
+        ltrFrame.setLocation(new Point(10, 10));
+        ltrFrame.setVisible(true);
+
+        rtlFrame = new JFrame("Right To Left");
+        contentPane = rtlFrame.getContentPane();
+        contentPane.setLayout(new BorderLayout());
+        panel = new JPanel();
+        panel.setBackground( Color.white );
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JToolBar/RightLeftOrientation.java:125]:
```suggestion
        panel.setBackground(Color.white);
```

**Review by kumarabhi006 — APPROVED** [review]
Few minor formatting changes else verified the test and it looks good to me.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**